QuestionsforKaida_s2.md

```diff
@@ -0,0 +1,6 @@
+- How can I refactor the code without repeating the  footer and nav for each page?
+- What is the best way to import google fonts? link or @url import 
+- How to target the right element? When should I target the parent and when should I target the child? sometimes it seems interchangeable
```

Comment:
Generally I like to do something like `.my-class li` or `#my-id ul`, where I make sure to have the specific name and then target into the children based on the elements.

That way you don't have to add many classes/ids to your code.  

Generally it makes sense to target the parent if you want _all_ its children to have a property but to target the child if you want one of them to be different.  

(For example, you probably want to set the font for a whole `<section>` but might want to make a specific paragraph _within_ that section have a different font so then you would target the child `p` tag.)  

Hope that helps!  Feel free to Slack me if you have more questions!